pkg/parser/paragraph_test.go

```diff
@@ -856,5 +856,91 @@ a foo image:foo.png[]`
 				Expect(ParseDocument(source)).To(MatchDocument(expected))
 			})
 		})
+
+		Context("list paragraphs", func() {
+
+			It("inline image within a quote", func() {
```

Comment:
I'll rename that test in #726, its title does not reflect the content of `source`